onnxruntime/wasm/api.cc

```diff
@@ -8,12 +8,6 @@
 #include <iostream>
 #include <vector>
 
-namespace {
-OrtEnv* g_env;
-OrtErrorCode g_last_error_code;
-std::string g_last_error_message;
-}  // namespace
```

Comment:
why do you remove them?

please keep global variables definitions in source files, and declaration only in header files. those are singletons that you don't want to have multiple of them in different objects

Reply:
Thanks, I originally moved this namespace so that the training API can access `g_env`. In the last commit, I named the namespace to `OrtGlobals` and added the declaration only to the header file, with the definition in api.cc, so that training and inference can share `g_last_error_code`. Let me know your thoughts / if there is a cleaner solution.